core/includes/install.core.inc

```diff
@@ -1714,6 +1714,11 @@ function install_finished(&$install_state) {
 
   $message = st('Thank you for installing @profile!', array('@profile' => backdrop_install_profile_distribution_name()));
```

Comment:
Since this is part of the generic installation code, not part of a profile, we probably want to maintain the messaging that uses a customizable distribution name -- though Backdrop doesn't really support distributions in any official capacity. Perhaps that should be a separate issue. In the mean time, we should probably keep using a variable rather than saying "Backdrop" specifically.

```suggestion
  $profile = backdrop_install_profile_distribution_name();
  $message = st('Thank you for installing @profile!', array('@profile' => $profile));
```